setup.py

```diff
@@ -41,6 +44,29 @@
     ],
 }
 
+
+class BuildPyCommand(setuptools.command.build_py.build_py):
+    """Custom build command for packaging."""
+
+    def _make_thrift(self):
+        """Generate Baseplate Thrift definitions.
+        """
+        # Make sure thrift executable exists.
+        thrift_check_cmd = ['/usr/bin/which', 'thrift1']
```

Comment:
why check for thrift1 but not make? any reason to not just run make and check status at the end without any up front checks?

Reply:
Ah, can and should do that as well! 

I just wanted to make the error message a bit more informative for each case; in the debian case you can define these dependencies so I wanted to try and echo that to an extent here.